core/planner.c

```diff
@@ -825,11 +825,12 @@ bool plan(struct deco_state *ds, struct diveplan *diveplan, struct dive *dive, i
 	if ((divemode == CCR || divemode == PSCR) && prefs.dobailout) {
 		divemode = OC;
 		po2 = 0;
+		int bailoutsegment = prefs.min_switch_duration < prefs.problemsolvingtime * 60 ? prefs.problemsolvingtime * 60 : prefs.min_switch_duration;
```

Comment:
What about
```
		int bailoutsegment = MAX(prefs.problemsolvingtime * 60, prefs.min_switch_duration);
```
to make this understandable without mental gymnastics?

[Completely unrelated: the things you have to do to define a proper MAX macro ... jeez. And that's probably not even standard C.]